### DISCUSSION TITLE
Recover/Edit an old Registry submission

### DISCUSSION THREAD

**Discussion by ivaneidel** [WalletConnect/walletconnect-monorepo#2325] [Developer Support] [answered]
I'd like to know if there's a way to claim ownership of an old Registry submission.

We have an wallet app submitted to the explorer (in fact is duplicated) and we'd like to remove one of them and update the other. The problem is that they both were submitted to the old registry and not the new Cloud Explorer, so we cannot access them to edit or delete them.

Is there any way to solve that?

- Our wallet name is **Defiant**.
- The published submissions are the ones with id: **ffa139f74d1c8ebbb748cf0166f92d886e8c81b521c2193aa940e00626f4e215** and **1986e7c874bb906f057d5d64a4806c004e021689536e5228c74d64a6058e8bac**

**Comment by finessevanes** [comment]
Let me get some more information about how we can help!

**Comment by finessevanes** [comment]
![Screenshot 2023-05-10 at 10 35 03 PM](https://github.com/WalletConnect/walletconnect-monorepo/assets/44731696/6044edfc-4074-4d29-b022-7450182bd7ad)
Are one of these Defiant wallets in question?

**Comment by ivaneidel** [comment]
Yes! All of them are the same!

That last one I'm not sure, but I assume is maybe one that we did from the cloud panel that we didn't know was there...?

**Comment by finessevanes** [comment]
Can you provide the UID or Project Id that you want to keep?

**Comment by ivaneidel** [comment]
Of course!
The one we'd like to keep is: **99f05650-2f5d-45fd-bf9f-76223151b10d**
It's still pending of review but the information is the same as the others with a few additions

**Reply by finessevanes** [reply]
Awesome- I'm p sure it's this one, but can you confirm this is the UID? 31019154-1d44-42e7-933c-71f9e25a97a5

**Reply by ivaneidel** [reply]
I'm sorry, I answered with the wrong id. The UUID is **99f05650-2f5d-45fd-bf9f-76223151b10d**

**Reply by finessevanes** [reply]
Weird, I just checked all 4 Defiant listings, (1 pending review, 2 published, 1 draft) and neither had that UID. Let me check with our explorer team.

**Reply by finessevanes** [reply]
We found your listing. Can you provide steps into how to check test for WalletConnect support? I'm not sure where to go from here: 

https://github.com/WalletConnect/walletconnect-monorepo/assets/44731696/e3deb622-7a4f-48c1-95bb-1a71aa4d4d73

**Reply by ivaneidel** [reply]
Of course!

You're using our Web3 InApp Browser there, it uses both an injected connector and our WalletConnect implementation underneath. But if you want to test just WalletConnect, you can go to Menu > WalletConnect > New Connection and use WC in the standard way with the QR code and such.

By the way, that site you accessed has a bit of complex javascript that sometimes return that blank state, we don't know why is that, but you can type in the Browser's search bar any URL you like and test there.

I must say though, we don't have yet support for WalletConnect V2, so it's just V1 for the moment, we're developing the V2 support but it's not production ready yet

**Reply by finessevanes** [reply, marked as answer]
Looks like your project has been published!
